content/workers/platform/routing/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+pcx-content-type: concept
+title: Routing
+---
+
+# Routing
+
+## Background
+
+Routing lets customers connect a Worker to the Internet, allowing it to receive HTTP requests on their Cloudflare zones. There are two ways to route to a Worker: 
+
+1. [Custom Domains](/workers/platform/routing/custom-domains).
+
+and
+
+2. [Routes](/workers/platform/routing/routes).
+
+Routes add Workers functionality to your existing proxied (orange-clouded) hostnames, as a proxy in front of your application server. 
+
+Custom Domains can replace the proxy (orange-cloud) process entirely. Custom Domains automatically attach a Worker to your hostname by creating a DNS record and an SSL certificate on your behalf. Custom Domains can also be invoked within the same zone via `fetch()`, unlike Routes.
```

Comment:
@kabirsikand are we actually provisioning certificates here? Afaik the certificate is already provisioned for the Cloudflare zone already.

Reply:
we are via a cert api